recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/arch/AttributeContainer.java

```diff
@@ -18,8 +18,7 @@ public List<Annotation> getAnnotations() {
 	}
 
 	public void setAnnotations(List<Annotation> annotations) {
-		this.annotations.clear();
-		this.annotations.addAll(annotations);
+		this.annotations = annotations;
```

Comment:
Why?

Reply:
You need to know the consequences, you cannot "listen" to annotations anymore. Like taking getAnnotations and having a reference only to that. Make sure that this is intended behavior.